pyproject.toml

```diff
@@ -11,31 +11,31 @@ readme = "README.md"
 requires-python = ">=3.7"
 license = { file = "LICENSE" }
 keywords = [
-  "automation",
-  "flake8",
-  "pycodestyle",
-  "pyflakes",
-  "pylint",
-  "clippy",
+    "automation",
```

Comment:
I was about to rever this change but we use 4 spaces according to our editorconfig.